src/utils.ts

```diff
@@ -93,4 +94,18 @@ export function makeRandomHexString(length) {
 		result += chars[idx];
 	}
 	return result;
-}
\ No newline at end of file
+}
+
+export function deleteNoFailRecursive(path:any) {
```

Comment:
Is it used only in the tests or I missearch?
If it is the case, maybe worth moving it to a specific file inside test sources. What do you think?